students/idcrickmore/lesson01/break_me.py

```diff
@@ -0,0 +1,22 @@
+#Lesson01 Break Me
+
+def name_error():
+	print(1+b)
+	
+def type_error():
+	print(1+"1")
+
+def syntax_error():
+	print 1+1
+
+def attribute_error():
+	not_a_list = 1
+	print(not_a_list.sort)
```

Comment:
check your editor settings, you want exactly 4 spaces for indentation

Reply:
also, you can configure your editor to link check your code or use online linter: http://pep8online.com/

this will enforce good python code styling